commands.go

```diff
@@ -28,8 +28,9 @@ const (
 	deleteUserMessageType = messaging.MessageType(0x32)
 
 	// Shard messages
-	createShardGroupIfNotExistsMessageType = messaging.MessageType(0x40)
-	deleteShardGroupMessageType            = messaging.MessageType(0x41)
+	createShardGroupIfNotExistsMessageType    = messaging.MessageType(0x40)
```

Comment:
pre-create happens after then initial create.  I'm not real happy with the naming.  One represents more of the "initial" creation now, and the other represents "ongoing" creation.